src/ompl/base/SpaceInformation.h

```diff
@@ -180,6 +180,11 @@ namespace ompl
                 return motionValidator_;
             }
 
+            MotionValidatorPtr& getMotionValidatorNonConst()
```

Comment:
Just change the method name to `getMotionValidator`. The right version of the method will be called automatically.